proto/src/main/proto/Rss.proto

```diff
@@ -391,6 +393,8 @@ message AppHeartBeatResponse {
 message ApplicationInfoRequest {
   string appId = 1;
   string user = 2;
+  optional string version = 3;
```

Comment:
Why  does ApplictionInfoRequest need `version` and `gitCommitId`?

Reply:
With version and git commit id, we can quickly get the client branch information, which can help us quickly locate the problem.